commerce-notification-api/src/main/java/com/liferay/commerce/notification/type/CommerceNotificationType.java

```diff
@@ -27,6 +27,10 @@
 @ProviderType
 public interface CommerceNotificationType {
```

Comment:
I would like to add the type: CommerceNotificationType<T> . so who implements don't need to cast